spec/controllers/moab_storage_controller_spec.rb

```diff
@@ -56,6 +57,22 @@
         get :show, params: { id: fixture_druid }
         expect(assigns(:output)[:current_version]).to eq 10
       end
+
+      it 'object_size' do
+        get :show, params: { id: fixture_druid }
+        expect(assigns(:output)[:object_size]).to be_between(21_900_000, 22_000_000)
+      end
+
+      it 'object_size_human' do
+        get :show, params: { id: fixture_druid }
+        expect(assigns(:output)[:object_size_human]).to include("MB")
+      end
+
+      it 'object_size_human instance' do
+        get :show, params: { id: fixture_druid }
+        expect(assigns(:output)[:object_size_human]).to be_kind_of String
```

Comment:
one last quibble -- I would just put this expects statement in the previous test.  And, to get picky, I think it could actually be

`expect(assigns(:output)[:object_size_human]).to be a_string_ending_with("MB")`

check out https://relishapp.com/rspec/rspec-expectations/v/3-6/docs/composing-matchers

Reply:
good suggestion.  i'd be inclined to just replace this and the `to include("MB")` test with the expectation naomi suggested.  that's a pretty neat composition -- i've done similar with expecting something `to be a_hash_including`, but i don't think i would've thought to do the `be a_string_ending_with` thing.